Makefile

```diff
@@ -311,7 +311,7 @@ out/minikube.iso: $(shell find "deploy/iso/minikube-iso" -type f)
 ifeq ($(IN_DOCKER),1)
 	$(MAKE) minikube_iso
 else
-	docker run --rm --workdir /mnt --volume $(CURDIR):/mnt $(ISO_DOCKER_EXTRA_ARGS) \
+	docker run --privileged --rm --workdir /mnt --volume $(CURDIR):/mnt $(ISO_DOCKER_EXTRA_ARGS) \
```

Comment:
why privileged ?

Reply:
Added `--privileged` only on suggestion that it might allow additional resources to the job and decrease execution time.

Will not submit the PR with `--privileged` flag.